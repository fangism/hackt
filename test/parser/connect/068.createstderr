FATAL: reference to uninstantiated process portaproc<> index [1]
ERROR: unrolling aliases.
ERROR: resolving super instance of port connection: p[1]
Error encountered during module::create.
ERROR in creating.  Aborting.
