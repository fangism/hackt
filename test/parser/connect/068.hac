// "068.hac" should fail: not instantiated yet

defproc portaproc(bool b, c[4]; int i, j[2]) { }

bool a;
bool x[4];
int q;
int r[2];

portaproc p[0..0];
p[0](a, x, q, r);
p[1]( ,  ,  ,  );	// not instantiated yet

