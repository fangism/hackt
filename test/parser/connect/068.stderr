ERROR: The following referenced indices of "p" have definitely not been instantiated: {
[1]
} [12:3]
